shared/security/src/main/java/io/pravega/shared/security/auth/AccessOperation.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.shared.security.auth;
+
+/**
+ * Represents an operation that is authorized by a permission (allow, deny, etc.). Currently, permissions
+ * always 'allow' access to specified access operations in Pravega.
+ */
+public enum AccessOperation {
+    /**
+     * Not known.
+     */
+    UNKNOWN,
+
+    /**
+     * Any operation.
```

Comment:
Clearly define what the behavior of this is. What is the most restrictive level that this implies?

Consider documenting the same for the other ones too so that whomever uses these understands what to expect.

Reply:
I've modified the documentation in the class. Please see if it looks better now.